packages/client/src/app/services/user/notifications.service.ts

```diff
@@ -19,7 +18,7 @@ export class NotificationsService {
 
     private url = `/api/notifications`;
 
-    constructor(private http: HttpClient, private zone: NgZone) {
+    constructor(private networkService: NetworkService, private zone: NgZone) {
         this.currentUserSubscription = this.currentUser$.subscribe(x => {
             this.currentUser = x;
```

Comment:
I left this alive, because it maintains a bit of state, and can serve as a wrapper around its network calls--I can see it doing more stuff in the future, too.